projects/plugins/jetpack/modules/search/customberg/components/sidebar/excluded-post-types-control.jsx

```diff
@@ -0,0 +1,59 @@
+/**
+ * WordPress dependencies
+ */
+import { useMemo } from '@wordpress/element';
+import { CheckboxControl, Notice } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/* eslint-disable react/jsx-no-bind */
+
+const VALID_POST_TYPES = global.JetpackInstantSearchOptions.postTypes;
```

Comment:
Could we use it as params or in state store or anything, using globals directly is pretty unfriendly for unit testing.

Reply:
Converted to a component property with the global value as a default, which should enable straightforward unit testing.